File: ruby/day6/instructor/fizzbuzz.rb
result = []

for i in 1..100
  if i % 3 == 0 && i % 5 == 0
    result << "FIZZBUZZ"
  elsif i % 3 == 0
    result << "FIZZ"
  elsif i % 5 == 0
    result << "BUZZ"
  else
    result << i
  end
end

p result

File: javascript/day2/assignments/fizzbuzz.js
let number = 1;
let first = Math.floor(Math.random() * 100);
let second = Math.floor(Math.random() * 100);

console.log(`first: ${first}\nsecond: ${second}`);

while(number <=100){
  if (number%first === 0 && number%second === 0) {
    console.log('fizzbuzz');
  } else if (number%first === 0){
    console.log('fizz');
  } else if(number%second === 0) {
    console.log('buzz');
  } else {
    console.log(number);
  }

  number++;
}
